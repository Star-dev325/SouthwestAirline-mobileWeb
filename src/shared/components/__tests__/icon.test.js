import { render } from '@testing-library/react';
import React from 'react';
import Icon from 'src/shared/components/icon';

describe('Icon', () => {
  it('Icon should render when with prop `type` ', () => {
    const { container } = render(<Icon type="cloud" />);

    expect(container).toMatchSnapshot();
  });
});
