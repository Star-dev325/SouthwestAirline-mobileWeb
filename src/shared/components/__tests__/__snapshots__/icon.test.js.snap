// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Icon should render when with prop \`type\`  1`] = `
<div>
  <i
    class="icon icon_cloud"
  />
</div>
`;
